### PR TITLE
fix: ci runs paths, turbo, husky, eslint

### PR DESCRIPTION
**Pull request by marlonmarcello**
# Description

Fixes CI paths, husky, and eslint migration to v9

### CODE REVIEW

File: packages/wethegit-react-hooks/eslint.config.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      },
+
+      parser: tsParser,
+      ecmaVersion: "latest",
+      sourceType: "module",
+    },
+
+    settings: {
+      react: {
+        version: "17.2",
```
Comment (andrewrubin) [inline packages/wethegit-react-hooks/eslint.config.js:62]: Should this be v18 @marlonmarcello ?

Comment (andrewrubin) [reply packages/wethegit-react-hooks/eslint.config.js:62]: Also looks like there's this `detect` option we might be able to use

```
    "react": {
      "version": "detect"
    }
```

**Review by NicholasLancey — COMMENTED** [review]
Tested this one locally and all looks good to me, will wait until Andrews comment is addressed before approving!

File: packages/wethegit-react-hooks/eslint.config.js (hunk 1 above)
Comment (marlonmarcello) [reply packages/wethegit-react-hooks/eslint.config.js]: That's fixed, thanks @andrewrubin !

**Review by andrewrubin — APPROVED** [review]

**Pull request merged**